src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.*;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+
+public class HomePage extends AbstractPage {
+
+    private final By signInBtnLocator = By.linkText("Sign In");
+    private final By topsLinkLocator = By.linkText("Tops");
+    private final By womenLinkLocator = By.xpath("//nav[@class='navigation']//span[contains(text(), 'Women')]");
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Step("Going to Registration Page")
+    public void openSignInPage() {
+        driver.findElement(signInBtnLocator).click();
+    }
+
+    @Step("Going to Women Tops Page")
+    public void clickWomenTops() {
+        Actions actions = new Actions(driver);
+        actions.moveToElement(driver.findElement(womenLinkLocator)).perform();
+        driver.findElement(topsLinkLocator).click();
```

Comment:
when you are using selenium it's better to create in your AbstractPage wrapper methods for clicking element, waiting for element to be visible and so on.
The method for clicking on element ideally should do the following:
First, wait for the element  to be visible,
then wait for the element to be clickable,
only then click on the element.

Reply:
I already had a method that would wait for a specific element on in AbstractPage class, but forgot to use it here. Changed it to a clickElement method with the waits that you specified